omod-1.9/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_9/PersonController1_9Test.java

```diff
@@ -82,6 +82,23 @@ public void shouldGetAPersonByUuid() throws Exception {
 		assertNull(PropertyUtils.getProperty(result, "auditInfo"));
 	}
 	
+	@Test
+	public void shouldGetALinkIfPersonIsPatient() throws Exception {
+		MockHttpServletRequest req = request(RequestMethod.GET, getURI() + "/" + getUuid());
+		SimpleObject result = deserialize(handle(req));
+		
+		List<Object> links = (List<Object>) result.get("links");
+		String expectedLink = RestConstants.URI_PREFIX + "v1/patient/" + RestTestConstants1_8.PATIENT_UUID;
+		String recievedLink = null;
+		for (Object link : links.toArray()) {
+			if ("patient".equals((String) PropertyUtils.getProperty(link, "rel"))) {
+				recievedLink = (String) PropertyUtils.getProperty(link, "uri");
+			}
+		}
+		assertNotNull(recievedLink);
+		assertEquals(expectedLink, recievedLink);
+	}
+	
```

Comment:
Any reason for putting this test in the omod-1.9 instead of omod-1.8?

Reply:
All other tests on other functionalities are placed here. Same convention is followed by other test suites to other resources. Is there any reason not follow this?